llvm/lib/Target/AIE/AIE2TargetTransformInfo.cpp

```diff
@@ -54,6 +54,9 @@ static cl::opt<int>
 static cl::opt<unsigned>
     MaxUnrollCost("aie-unroll-max-cost", cl::Hidden, cl::init(200),
                   cl::desc("Maximum partial unroll cost for loops"));
+static cl::opt<unsigned> PreferSwpOverUnroll(
+    "aie-prefer-swp-over-unroll", cl::Hidden, cl::init(9),
+    cl::desc("Aim for pipelining if MinIterCount is at least this value."));
```

Comment:
The default is inspired by the fact that all ReduceABC's regress for 8 and lower.